types/bull-arena/index.d.ts

```diff
@@ -3,12 +3,13 @@
 // Definitions by: Levi Bostian <https://github.com/levibostian>
 //                 Gaurav Sharma <https://github.com/gtpan77>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.8
+// TypeScript Version: 4.0
```

Comment:
What is the reason for bumping this up?

Reply:
will revert

Reply:
Build is failing when I reverted to 2.8
```
Accessors are only available when targeting ECMAScript 5 and higher.
```

Reply:
CI build was failing on 2.8. When I changed it to 4.0, build passed without any problem. I guess bullmq is using latest version of typescript. I am not familiar with Accessors.